yarn.lock

```diff
@@ -5905,6 +5905,7 @@ __metadata:
     react-focus-lock: ^2.9.5
     react-modal: ^3.16.1
     react-onclickoutside: ^6.13.0
+    react-select: ^5.7.4
```

Comment:
Can the existing dependency in the terra-ui section be removed then?

Reply:
My only concern is, i do not know the repercussions of it. Should i make the change just on the basis of unit tests or is there a better way of testing that i am not breaking anything

Reply:
For what it's worth, `SingleValue` is a _type_. Changing types doesn't affect the behavior of the app. So I think it's fine to rely on a passing build (specifically its `tsc` step) to verify the change.

Reply:
I removed some of SingleValue, others were giving error as it was directly passing value to onChange for select (AzurePersistentDiskSizeSelect/PersistentDiskTypeSelect). I moved them int o packages.

Reply:
Reverted it, will it be ok to push in just Select PR and i can have a follow up to remove react-select. Doing that can start the work in Workbench to use terra-ui select

Reply:
Removed the dependency from Terra UI in https://github.com/DataBiosphere/terra-ui/pull/4713.